circuits/benchmarks/scripts/generate_report.sh

```
@@ -226,6 +415,35 @@ IFS='|' read -r p3t p3s p3b <<< "$p3"
+fi
+p4a_integration=$(integration_timing_seconds "Ciphertext published -> PlaintextAggregated")
+if [ -n "$p4a_integration" ] && [ "$p4a_integration" != "null" ]; then
+    p4at="$(format_s "$p4a_integration") s"
+fi
+
+# Keep role-phase rows aligned with artifact outputs when folded artifact sizes are available.
+p2_artifact=$(artifact_size_pair_from_gas "dkg")
+if [ -n "$p2_artifact" ]; then
+    IFS='|' read -r p2s p2b <<< "$p2_artifact"
+fi
+p3_artifact_proof=""
```

Shellcheck (0.11.0)

[warning] p3_artifact_proof appears unused. Verify use (or export if used externally).

(SC2034)